File: examples/js/updateAnomaly.js
const anomalyId = 'YOUR_ANOMALY_ID';
const authToken = process.env.AIOPS_TOKEN || 'YOUR_TOKEN';
// 示例：Update anomaly
fetch(`http://localhost:8000/api/v1/anomalies/${anomalyId}`, {
    method: 'PUT',
    headers: { Authorization: `Bearer ${authToken}` }
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.error('Request error:', err));

File: examples/python/updateAnomaly.py
# -*- coding: utf-8 -*-
import requests

YOUR_ANOMALY_ID = ""
anomaly_id = "YOUR_ANOMALY_ID"

# 示例：Update anomaly
# 使用 PUT 方法请求 /anomalies/{anomaly_id}
url = f"http://localhost:8000/api/v1/anomalies/{anomaly_id}"
headers = {"Authorization": "Bearer YOUR_TOKEN"}

try:
    response = requests.put(url, headers=headers)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print("Request failed:", e)
